spyder_line_profiler/spyder/widgets.py

```diff
@@ -19,41 +16,34 @@
 import os
 import os.path as osp
 import time
-import sys
+from datetime import datetime
 
 # Third party imports
-from qtpy.compat import getopenfilename
-from qtpy.QtCore import (QByteArray, QProcess, Qt, QTextCodec,
-                         QProcessEnvironment, Signal)
+from qtpy.QtWidgets import QHBoxLayout, QLabel
 from qtpy.QtGui import QBrush, QColor, QFont
-from qtpy.QtWidgets import (QHBoxLayout, QWidget, QMessageBox, QVBoxLayout,
-                            QLabel, QTreeWidget, QTreeWidgetItem, QApplication)
+from qtpy.QtCore import (QByteArray, QProcess, Qt, QTextCodec,
+                         QProcessEnvironment, Signal, QTimer)
+from qtpy.QtWidgets import (QMessageBox, QVBoxLayout,
```

Comment:
`QLabel` import from the removed line:
```suggestion
from qtpy.QtWidgets import (QMessageBox, QVBoxLayout, QLabel,
```